app/views/provisioning_templates/welcome.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for(:title, _("Provisioning Templates")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("info-circle", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Provisioning Templates') %></h1>
+  <p><%= _("The Provision Templates is the core of Foreman’s flexibility to deploy the right OS with the right options.
```

Comment:
This doesn't make much sense, I'm not even sure what the point is. Can you rewrite it, perhaps to say that provisioning templates are used for provisioning and installation of operating systems?